src/vma/dev/qp_mgr_eth_mlx5.h

```diff
@@ -42,6 +42,23 @@
 
 #include <infiniband/mlx5_hw.h>
 
+#if !defined(MLX5_ETH_INLINE_HEADER_SIZE)
```

Comment:
Mellanox OFED 3.3 does not have some needed types that was used for socketxtreme mode during development. VMA started socketxtreme support since MOFED 3.4.

Reply:
i understand the logic.
anyway, it should be located in the cpp file